lib/schedule/health_server.ex

```diff
@@ -47,6 +48,11 @@ defmodule Schedule.HealthServer do
 
   @impl true
   def handle_call(:ready?, _from, state) do
-    {:reply, state == :loaded, state}
+    checker_healthy_fn =
+      Application.get_env(:skate, :checker_healthy_fn, &Schedule.Health.Checker.healthy?/0)
+
+    healthy? = state == :loaded && checker_healthy_fn.()
```

Comment:
Hm, yeah. So just thinking it through, once we we are loaded our data might not be healthy. But once it is, it will stay healthy (at least in this world right now where we never reload GTFS), so thus we shouldn't need to run the checks again. 

Calling it once from the `loaded` call is a good approach. We would probably need to change the `loaded` state value to `healthy` and `unhealthy` values.

Is there any reason the data might not be responding fully yet, or might go from unhealthy to healthy? I haven't thought of any yet, but if so, we could instead add a `healthy` state value that could come after `loaded` and once set bypass the `Checker` call in `ready?`.

Reply:
Maybe if there's a lot of pent-up requests from the realtime system, and the server takes so long handling those that the healthy check times out? I think that's unlikely, and I think calling the server from loaded() is fine.